benchmark/distributed/master.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+const jstp = require('../..');
+
+const yargs = require('yargs');
+const fs = require('fs');
+const path = require('path');
+const parseAddress = require('./parse-address');
+const statistics = require('../statistics');
```

Comment:
I'd to suggest to split local files from npm packets via empty line as we have in `lib`. I've checked the other benchmark and it also has it all in one place, so I think if we agree on this one we should change it there too.